### PR TITLE
Implements #35 Solution 1

### PR DESCRIPTION
**Pull request by papr**

### CODE REVIEW

**Comment by cpicanco** [comment]
@papr , tested and working smoothly.

File: pyglui/gldraw.pxi [hunk 1]
```diff
@@ -32,7 +32,7 @@ cdef inline rect_corners(Vec2 org, Vec2 end, RGBA color):
     gl.glVertex3f(end.x,org.y,0.0)
     gl.glEnd()
 
-cdef inline tripple_h(Vec2 org, Vec2 size):
```
Comment (mkassner) [inline pyglui/gldraw.pxi]: can these be turned back into cdefs? It faster and this is used a lot.

Comment (mkassner) [inline pyglui/gldraw.pxi]: you can redefine the the input to fotbox type if that works.

File: pyglui/menus.pxi [hunk 2]
```diff
@@ -582,21 +586,22 @@ cdef class Scrolling_Menu(Base_Menu):
             #mouse:
             self.scrollbar.handle_input(new_input,visible)
             #scrollwheel:
-            if new_input.s.y and visible and self.element_space.mouse_over(new_input.m):
+            if new_input.s.y and visible and self.element_space.mouse_over(new_input.mouse_pos):
```
Comment (mkassner) [inline pyglui/menus.pxi]: while more legible, we want this to be fast. Can you remove the mouse_pos property and use m OR rename m to mouse_pos everywhere?

File: pyglui/ui.pyx [hunk 3]
```diff
@@ -715,3 +720,6 @@ cdef class Vec2:
                 self.y == obj
             else:
                 raise IndexError()
+
+    cpdef bint is_contained_in_rect(self, Vec2 rect_org, Vec2 rect_size):
```
Comment (mkassner) [inline pyglui/ui.pyx]: make this be a cdef only.

**Comment by mkassner** [comment]
Actually please ignore my comments. I think we should implement this differently.

**Pull request closed without merging**